src/SearchButton.js

```diff
@@ -0,0 +1,5 @@
+import react from "react";
+
+const Button = () => <button>Search</button>;
+
+export default Button;
```

Comment:
FYI, The orginal button did have some classes on it that the Bootstrap library used to style it nicely, not that it really matters